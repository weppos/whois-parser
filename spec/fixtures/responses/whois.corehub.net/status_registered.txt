Domain Name: corehub.net
Registry Domain ID: D-179116748-CONE
Registrar WHOIS Server: whois.corehub.net
Registrar URL: http://corehub.net
Creation Date: 2012-07-14T16:04:42.0Z
Registrar Registration Expiration Date: 2018-07-14T16:04:42.0Z
Registrar: CORE-901 (Axone/Secretariat Maintenance Account)
Registrar IANA ID: 15
Registrar Abuse Contact Email: abuse@corehub.net
Registrar Abuse Contact Phone: +34.935275235
Domain Status: ok    https://www.icann.org/epp#ok
Registry Registrant ID: COCO-1883998
Registrant Name: CORE ASSOCIATION
Registrant Organization: 
Registrant Street: 
Registrant City: Geneva
Registrant State/Province: GE
Registrant Postal Code: CH-1215
Registrant Country: CH
Registrant Phone: +41.229295744
Registrant Phone Ext: 
Registrant Fax: +41.229295745
Registrant Fax Ext: 
Registrant Email: secretariat@corenic.org
Registry Admin ID: COCO-354
Admin Name: CORE Secretariat
Admin Organization: CORE Association
Admin Street: cour de Rive 2
Admin City: Geneve
Admin State/Province: GE
Admin Postal Code: 1204
Admin Country: CH
Admin Phone: +41.223125610
Admin Phone Ext: 
Admin Fax: 
Admin Fax Ext: 
Admin Email: secretariat@corenic.org
Registry Tech ID: COCO-354
Tech Name: CORE Secretariat
Tech Organization: CORE Association
Tech Street: cour de Rive 2
Tech City: Geneve
Tech State/Province: GE
Tech Postal Code: 1204
Tech Country: CH
Tech Phone: +41.223125610
Tech Phone Ext: 
Tech Fax: 
Tech Fax Ext: 
Tech Email: secretariat@corenic.org
Name Server: us20-west.irondns.net 104.225.5.66 2607:fc50:2000:2001::2
Name Server: hk21.irondns.net 2403:2500:8000:1::9f9 103.6.86.97
Name Server: us19-east.irondns.net 4.71.215.210 2001:1900:2236::d2
DNSSEC: unsigned
URL of the ICANN WHOIS Data Problem Reporting System: http://wdprs.internic.net
>>> Last update of WHOIS database: 2018-05-03T13:22:13.239Z <<<

% (c) 2014 COREhub S.R.L.U. ("COREhub"). All rights reserved.
%
% NOTICE: Access to the domains information is provided to assist in
% determining the contents of a domain name registration record in the
% COREhub database. The data in this record is provided by COREhub for
% informational purposes only, and COREhub does not guarantee its
% accuracy. This service is intended only for query-based access. You
% are not authorized to access or query our Whois database through the
% use of electronic processes that are high-volume and automated except
% as reasonably necessary to register domain names or modify existing
% registrations.
%
% By submitting a Whois query, you agree that you will use this data
% only for lawful purposes and that, under no circumstances, will you
% use this data (i) to allow, enable, or otherwise support the
% transmission by e-mail, telephone, postal mail, facsimile or other
% means of mass unsolicited, commercial advertising or solicitations to
% entities other than the data recipient's own existing customers;
% and/or (ii) to enable high volume, automated, electronic processes
% that send queries or data to the COREhub systems, except as reasonably
% necessary to register domain names or modify existing registrations.
%
% The compilation, repackaging, dissemination or other use of the data
% in the COREhub Whois database is expressly prohibited without the
% prior written consent of COREhub. COREhub reserves the right to
% restrict your access to the Whois database in its sole discretion to
% ensure operational stability. COREhub may restrict or terminate your
% access to the Whois database for failure to abide by these terms of
% use. COREhub reserves the right to modify these terms at any time.
%
% For more information on Whois status codes, please visit
% https://icann.org/epp
