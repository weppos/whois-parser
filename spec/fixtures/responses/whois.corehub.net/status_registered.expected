#domain
  %s == "corehub.net"

#domain_id
  %s == "D-179116748-CONE"

#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true

#created_on
  %s %CLASS{time}
  %s %TIME{2012-07-14 16:04:42 UTC}

#expires_on
  %s %CLASS{time}
  %s %TIME{2018-07-14 16:04:42 UTC}

#registrar
  %s %CLASS{registrar}
  %s.id           == "15"
  %s.name         == "CORE-901 (Axone/Secretariat Maintenance Account)"
  %s.organization == "CORE-901 (Axone/Secretariat Maintenance Account)"
  %s.url          == "http://corehub.net"

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].name          == "CORE ASSOCIATION"
  %s[0].organization  == ""
  %s[0].address       == ""
  %s[0].city          == "Geneva"
  %s[0].zip           == "CH-1215"
  %s[0].state         == "GE"
  %s[0].country_code  == "CH"
  %s[0].phone         == "+41.229295744"
  %s[0].fax           == "+41.229295745"
  %s[0].email         == "secretariat@corenic.org"

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_ADMINISTRATIVE
  %s[0].name          == "CORE Secretariat"
  %s[0].organization  == "CORE Association"
  %s[0].address       == "cour de Rive 2"
  %s[0].city          == "Geneve"
  %s[0].zip           == "1204"
  %s[0].state         == "GE"
  %s[0].country_code  == "CH"
  %s[0].phone         == "+41.223125610"
  %s[0].fax           == ""
  %s[0].email         == "secretariat@corenic.org"

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_TECHNICAL
  %s[0].name          == "CORE Secretariat"
  %s[0].organization  == "CORE Association"
  %s[0].address       == "cour de Rive 2"
  %s[0].city          == "Geneve"
  %s[0].zip           == "1204"
  %s[0].state         == "GE"
  %s[0].country_code  == "CH"
  %s[0].phone         == "+41.223125610"
  %s[0].fax           == ""
  %s[0].email         == "secretariat@corenic.org"

#nameservers
  %s %CLASS{array}
  %s %SIZE{3}
  %s[0] %CLASS{nameserver}
  %s[0].name == "us20-west.irondns.net"
  %s[0].ipv4 == "104.225.5.66"
  %s[0].ipv6 == "2607:fc50:2000:2001::2"
  %s[1] %CLASS{nameserver}
  %s[1].name == "hk21.irondns.net"
  %s[1].ipv4 == "103.6.86.97"
  %s[1].ipv6 == "2403:2500:8000:1::9f9"
  %s[2] %CLASS{nameserver}
  %s[2].name == "us19-east.irondns.net"
  %s[2].ipv4 == "4.71.215.210"
  %s[2].ipv6 == "2001:1900:2236::d2"
